platform/osx/detect.py

```diff
@@ -78,14 +78,16 @@ def configure(env):
         env["osxcross"] = True
 
     if env["arch"] == "arm64":
-        print("Building for macOS 10.15+, platform arm64.")
-        env.Append(CCFLAGS=["-arch", "arm64", "-mmacosx-version-min=10.15"])
-        env.Append(LINKFLAGS=["-arch", "arm64", "-mmacosx-version-min=10.15"])
+        print("Building for macOS 11.00+, platform arm64.")
+        env.Append(CCFLAGS=["-arch", "arm64", "-mmacosx-version-min=11.00"])
+        env.Append(LINKFLAGS=["-arch", "arm64", "-mmacosx-version-min=11.00"])
```

Comment:
Note: This is not a real change, 10.15 for ARM was never released, only a few early test builds for DTK were marked as 10.15.